### ISSUE TITLE
Add new task button not showing textarea after adding condition

### ISSUE THREAD

**Issue by raknos13** [raknos13/planero-planner-react#2]
New task button functionality not working as expected after adding new state `isAddingNewTask` to show the textarea only after user has clicked on `Add new task` button.

**Comment by raknos13** [comment]
[927d439](https://github.com/raknos13/planero-planner-react/commit/927d4391fa1fa35f5a145a2114dd86ac03b6eaa8) Fixes the issue. Closing.
